staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -935,6 +935,7 @@ const (
 	MigrationFailed VirtualMachineInstanceMigrationPhase = "Failed"
 )
 
+// VirtualMachineInstancePreset allows to modify VMI domain for VMIs that match provided label selector
```

Comment:
VirtualMachineInstancePreset defines a VMI spec.domain to be applied to all VMIs that match the provided label selector

But what happens when this is applied to a partial VMI definition ? What has precedence?

Reply:
> But what happens when this is applied to a partial VMI definition ? What has precedence?

it depends on how many presets match a VMI, ordering that they are applied, and what's in the original VMI. 
https://kubevirt.io/user-guide/virtual_machines/presets/#overrides

it can get complicated to understand fully and it isn't simple to document in it's entirety in a description. fyi, pod presets, which is what vmi presets are modeled after, have been deprecated do to this complexity and non-deterministic behavior.

Reply:
I modified the description and added the link as I find it useful.